DependencyInjection/Configuration.php

```diff
@@ -29,6 +29,23 @@
 final class Configuration implements ConfigurationInterface
 {
     /**
+     * kernel.debug param value.
```

Comment:
This is an implementation detail which is only true when used in the extension class. We can just use something like "Whether or not to enable the debug mode".